src/name.rs

```diff
@@ -20,6 +20,8 @@ use untrusted;
 #[cfg(feature = "std")]
 use std::string::String;
 
+use std::fmt;
```

Comment:
Let's not use `use std::fmt`; instead let's fully-qualify all the names that use `std`.

Reply:
k